test/api/search.test.js

```diff
@@ -141,6 +141,19 @@ describe('Search', () => {
           });
       });
     });
+    
+    describe('navigator is disabled', () => {
+      it('should have near undefined and return an autocomplete response', () => {
+        httpStub.resolves(autocompleteResponse);
+        
+        const disableNavigator = true;
+        return Search.autocomplete({ query, disableNavigator })
+          .then((response) => {
+            sinon.assert.calledWith(httpStub, 'GET', 'v1/search/autocomplete', { query: 'mock-query', near: undefined, limit: undefined });
```

Comment:
nit: can also make this assertion some form of `expect(httpStub).to.be.calledWith` - I believe we do so with some of the cookie / device tests

Reply:
ah nice yeah thanks ill follow the pattern